stripe-core/build.gradle

```diff
@@ -18,6 +18,8 @@ dependencies {
     implementation libs.kotlin.coroutinesAndroid
     implementation libs.kotlin.serialization
 
+    compileOnly libs.androidx.workManager
```

Comment:
Using `compileOnly` for now, so that we just piggyback off of any WorkManager dependency of the host app. If we’re feeling comfortable pulling in WorkManager ourselves, then I can change this to an `implementation`.

Reply:
How much of a size diff would this be?

Reply:
just clarifying here, compileOnly does not add the dependency. If it's present on the classpath it means it was added by the host app, then it can be used. 

We can switch to `implementation` and check the size increase on example apps, but it should not be a big deal. Version clashes between SDK and host app might be our primary concern in case of `implement`ing it on our side.